app/timetables/models.py

```diff
@@ -123,6 +123,27 @@ def save(self, *args, **kwargs):
     def __str__(self):
         return self.name
 
+    def calculate_cycle_day(self, date_datetime):
+        days_interval = (date_datetime - self.ref_cycle_date).days
+
+        if days_interval < 0:
+            cycle_day = None
+        else:
+            cycle_day = ((days_interval % self.cycle_length) + self.ref_cycle_day) \
```

Comment:
I don't know if we have any in-house Taverna coding convection against using backslashes for wrapping long lines - but a lot of developers frown against using this as it can easily bring about bugs if not used carefully, besides, it kinda makes your code less pretty :). Also, PEP 8 recommends wrapping expressions in parentheses over using backslashes. 

It might be worth deciding what style is ok to use.

What do you guys think ? cc @andela-kanyanwu @andela-uawili @allThePythonistasInTheHouse